src/System Application/App/Json/app.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
This code is already part of a separate PR.

Reply:
Yes, but how should i separate those?

Reply:
Thinking it through, I guess the easiest is that we get 716 and 719 processed. Once they're in, you can remove them from this DRAFT - and eventually take it out of DRAFT mode. There's a reason it's a DRAFT, right ;-)